babycenter/templates/core/article_page.html

```diff
@@ -22,7 +22,8 @@ <h2 class="l-indent heading heading--back heading{{section.get_effective_extra_s
   <h1 class="article__title">{{self.title}}</h1>
 
   {% if self.image %}
-  {% image self.image width-288 class="article__image" %}
+    {% image self.image width-288 as article_image %}
+    <img src="{{ article_image.url }}" class="article__image" />
```

Comment:
ahh...yes, those. Those are images within the body of the article. We'll need to test these out and see if they also suffer from this. I suspect they do.